README.md

```diff
@@ -34,26 +34,28 @@ Currently, OpenCensus supports:
 * [Jaeger][exporter-jaeger] for traces
 * [AWS X-Ray][exporter-xray] for traces
 
-## Tags
 
-Tags represent propagated key-value pairs. They are propagated using context.Context
-in the same process or can be encoded to be transmitted on the wire and decoded back
-to a tag.Map at the destination.
+## Overview
 
-### Getting a key by a name
+![OpenCensus Overview](https://i.imgur.com/cf4ElHE.jpg)
 
-A key is defined by its name. To use a key, a user needs to know its name and type.
+In a microservices environment, a user request may go through
+multiple services until there is a response. OpenCensus allows
+you to instrument your services and collect diagnostics data all
+through your services end-to-end.
 
-[embedmd]:# (tags.go stringKey)
-```go
-// Get a key to represent user OS.
-key, err := tag.NewKey("my.org/keys/user-os")
-if err != nil {
-	log.Fatal(err)
-}
-```
+Start with instrumenting HTTP and gRPC clients and servers,
+then add additional custom instrumentation.
+
+* [HTTP guide](https://github.com/census-instrumentation/opencensus-go/tree/master/examples/http)
```

Comment:
Would relative URLs like `(./examples/http)` suffice instead of the absolute hard coded one?

Reply:
IMO, it is better to keep them as they are. They still work from preview pages such as https://github.com/rakyll/opencensus-go/blob/200da9489b3e88e6494c0376ab758462214a826c/README.md.